tests/unit_tests/data_steward/validation/participants/identity_match_test.py

```diff
@@ -410,8 +411,8 @@ def test_match_participants_same_participant_simulate_location_pii_read_errors(
 
     def test_match_participants_same_participant_simulate_pii_read_errors(self):
         # pre conditions
-        self.mock_pii_values.side_effect = googleapiclient.errors.HttpError(
-            500, b'bar', b'baz')
+        self.mock_pii_values.side_effect = test_util.mock_google_http_error(
+            status_code=500, content=b'baz', reason='bar')
```

Comment:
why change this mock to use a test_util object instead of an actual expected error?

Reply:
That test util func returns an actual object that is constructed with a `requests.Response` type as is now required.

It was previous not required, and I did not want to manually build an `HttpError` instance _and_ a `requests.Response` instance every time.